language-design.md

```diff
@@ -18,6 +18,53 @@ For each quote, describe what it was about the quote that led you pick it.
 **Response**
 
 
+1. "Many brilliant and talented people have contributed to R over the years, many
+of whom are renowned computer scientists for whom I have tremendous respect!
+But a computer scientist is not the same as a software engineer." [Smith, 2016]
+
+We picked this quote because while most of what we have been reading has been 
+from professors and researchers who study the design and implementation of 
+languages, we rarely get a voice from an "outsider" in this respect - someone 
+who just wants to use a language for a purpose and not to analyze its history 
+and design decisions. We think the goal of a well-designed and "good" DSL is to 
+be a tool that accomplishes the domain-specific goals easily or intuitively. 
+Smith does not think R is not well-designed, and traces the cause to the lack 
+of software engineers contributing to R.
+
+The author specifically makes the distinction between computer scientist and 
+software engineers, which brings up differences between academia and industry. 
+However, many languages developed for and used by academia are very planned 
+and are implemented very specifically, i.e. Prolog and Lisp. Languages 
+that are considered better designed than R also have a lot of contributions 
+from computer scientists who are not software engineers, so it doesn't seem 
+like that is the reason why R is badly designed.
+
+Doing some research, it seems that R was derived from S, with some language 
+components inspired by Scheme. It was developed by two people initially, and 
+now its design is directed by the R-core of around twenty people. It is unclear 
+how many of those people still actively contribute, and maybe that is the 
+problem. Maybe because R is open-source, a kind of a project built as a bazaar, 
+it incorporated too many different designs and features that, when looked at 
+together by a newcomer, seems like a series of hacks and a mishmash of things. 
```

Comment:
I agree that a "bazaar" design risks cohesiveness in favor of fast development. But I feel like a small core of decision-makers can mitigate this since they oversee the language, especially if some people are inactive. I don't know much about R though.